libpay-stripe/src/test/scala/com/wix/pay/stripe/StripeAdditionalInfoMapperTest.scala

```diff
@@ -1,31 +0,0 @@
-package com.wix.pay.stripe
```

Comment:
Why did you delete a test and didn't add the new one?

Reply:
i've lost them after a set of rebases / ammends
restored from local history + added refactoring
thanks!